src/test/kotlin/reactor/kotlin/test/StepVerifierExtensionsTests.kt

```diff
@@ -53,4 +58,49 @@ class StepVerifierExtensionsTests {
                 .verifyError<IllegalStateException>()
     }
 
+    @Test
+    fun `testUsingVirtualTime()`() {
+        { Mono.just("foo").delayElement(Duration.ofDays(1)) }
```

Comment:
maybe, if anything goes wrong, use `Duration.ofMinutes(10)` or something long enough that vts is significant, but if somehow it ends up using realtime the CI won't get stuck for too long?